models/three-color3values/README.md

```diff
@@ -0,0 +1,7 @@
+* Roblox Studio version: 0.438.0.407270
+
+Contains three `Color3Value`s, each with one of the following `Value`s:
+
+* `Color3.new(1, 2, 3)`
+* `Color3.new(4, 5, 6)`
+* `Color3.new(7, 8, 9)`
```

Comment:
Did you mean `Color3.fromRGB`? Can we have a mix of values across the range 0-255, and a couple beyond it (ie HDR colors?)